### ISSUE TITLE
Continuum fitting failure

### ISSUE THREAD

**Issue by LydiaMak** [legolason/PyQSOFit#34]
Hi,

I get some very weird results for the continuum fitting although the data don't seem weird in any sense. I attach the results with and without the polynomial term. Have you ever came across on stuff like these. Any suggestions?
![Screenshot 2022-05-15 at 13 18 26](https://user-images.githubusercontent.com/23122380/168468005-2cf3f887-ba46-41e2-a37b-9c1cd8affab0.png)
![Screenshot 2022-05-15 at 13 19 14](https://user-images.githubusercontent.com/23122380/168468010-59e8d615-23b0-4811-a3eb-5e19a51b3995.png)

Thanks,

Lydia

**Comment by legolason** [comment]
sometimes if some data points are extremely large will cause this issue, check the data.

**Comment by shiro1920** [comment]
Hi, I met the same question as you, have you solved this problem?

**Comment by LydiaMak** [comment]
@shiro1920 Not really. What I have found to be somehow useful is to change the windows in the PyQSOFit.py for the continuum fitting.

**Comment by burke86** [comment]
@LydiaMak If you have time and want to send me the spectrum and minimal code for how you are reading in the data/running the fitting I can look into it. You can find my email on my github user page.
